src/docfx/lib/HtmlUtility.cs

```diff
@@ -487,8 +487,14 @@ internal static void AddLinkType(ref HtmlToken token, string locale)
             }
         }
 
-        private static string AddLocaleIfMissing(string href, string locale)
+        private static string AddLocaleIfMissingForAbsolutePath(string href, string locale)
```

Comment:
```suggestion
        private static string PrependLocaleIfMissing(string href, string locale)
```

Reply:
Actually, this is not `prepend`.
And also, would like to specify this is for `AbosulutePath`.